### PR TITLE
[Wales] Add scraper/parser.

### PR DESCRIPTION
**Pull request by dracos**

### CODE REVIEW

File: pyscraper/wa/parse.py [hunk 1]
```diff
@@ -1,186 +1,370 @@
...
+
+    def parse_plenary(self, text):
+        soup = BeautifulSoup(text, 'xml')
+
+        self.speeches = []
+        current_agenda_id = '0'
+        for item in soup.find_all(['XML_Plenary_Bilingual', 'XML_Plenary-FifthSenedd_Bilingual']):
+            #entry_id = item.Contribution_ID
+            #tv_spoken = item.contribution_spoken_seneddTv
+            #tv_translated = item.contribution_translated_seneddTv
```
Comment (struan) [inline pyscraper/wa/parse.py:317]: I assume these are not needed?

**Review by struan — APPROVED** [review]
Works, all makes sense.

Some commented out code which I'm not sure if it's a TODO or not.

File: pyscraper/wa/parse.py (hunk 1 above)
Comment (dracos) [reply pyscraper/wa/parse.py:317]: Yeah, I left them in just as documentation on what was available in the XML, I guess.

**Comment by dracos** [comment]
@struan Sorry, have added a number of small fixups since initial try out - think ones that need look at are 0173f3460d1b24b40cd8f6655f08173afdcf1523 (mentioned in my comment above), b79202199fd13ba5d3de8b19cbe03c174b366e4d (in case QNR/votes change, but not plenary), 70e720a0ff837d24fb0878d4bc2286b990563b86 and 632689a48252d0a7426273663ada7eca5225a056 (if votes file reorders) from this morning.

**Pull request merged**